src/harness/renderers/gl/gl_renderer.c

```diff
@@ -220,6 +220,10 @@ void GLRenderer_Init(int pRender_width, int pRender_height) {
     LOG_INFO("OpenGL version string: %s", glGetString(GL_VERSION));
     LOG_INFO("OpenGL shading language version string: %s", glGetString(GL_SHADING_LANGUAGE_VERSION));
 
+    if (glGetString(GL_SHADING_LANGUAGE_VERSION) == NULL) {
```

Comment:
this resolves a null pointer error inside `glCreateProgram` later on inside my windows VM which doesn't expose gl shaders

Reply:
What OpenGL version does the vm expose?

But you're probably correct to test capabilities instead of requiring a minimum OpenGL version.